var/spack/repos/builtin/packages/opensubdiv/package.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Opensubdiv(CMakePackage, CudaPackage):
+    """OpenSubdiv is a set of open source libraries that
+    implement high performance subdivision surface (subdiv)
+    evaluation on massively parallel CPU and GPU architectures.
+    This code path is optimized for drawing deforming surfaces
+    with static topology at interactive framerates."""
+
+    homepage = "http://graphics.pixar.com/opensubdiv/docs/intro.html"
+    url      = "https://github.com/PixarAnimationStudios/OpenSubdiv/archive/v3_4_0.tar.gz"
+    git      = "https://github.com/PixarAnimationStudios/OpenSubdiv"
+
+    version('dev', branch='dev')
```

Comment:
You should call this version develop if you want it to sort properly

Reply:
fixed